stlab/channel.hpp

```diff
@@ -352,17 +388,24 @@ struct shared_process : shared_process_receiver<yield_type<T, Arg>>,
     }
 
     void clear_to_send() override {
+        // The downstream sender may trigger this because an other parallel upstream
+        // process may have just passed down a value
+        if (_process_suspend_count == 0)
```

Comment:
I don't see how - we should only cts when this process is suspended and the receiver queue now has an additional slot. This would imply a cts when not suspended and that would be a bug.